Master gradle

znck opened the pull request.

znck-graveyard committed:
Added .gitignore

znck-graveyard committed:
Switching to Gradle

znck-graveyard committed:
Project migrated to Gradle

znck-graveyard committed:
Gradle dependency added

znck-graveyard committed:
Gradle dependency added

znck-graveyard committed:
Gradle dependency added

znck-graveyard committed:
Modified build.gradle and Added couchdb.properties
- Added dependency - couchbase lite

znck-graveyard committed:
working on couchbase lite

znck-graveyard committed:
Modified .gitignore

znck-graveyard committed:
Pull from upstream

znck-graveyard committed:
Testing object store

jezekp commented:
I have created a NoSQL branch. Please, make this pull request into this branch. While the master that uses maven build is still under developlment it is difficult to merge it into the master now.

znck-graveyard committed:
Implemented NoSQL
 - Added Storage classes for Model
 - Added Storage Factory

 - #Mark-1 - Moving to run tests

Commit:
Merge pull request #1 from rahulkadyan/nosql

Merging NoSQL implementation branch to Gradle branch

znck-graveyard committed:
Added iml files to repository

znck-graveyard committed:
Merging with upstream
commit-id - 234d0fb04f4c4b044ff52cf20ef1ec9497ee2202 merged

znck-graveyard committed:
Updated dependencies

znck-graveyard committed:
Fixed errors in Store objects

znck-graveyard committed:
-- sat

znck-graveyard committed:
Updated JavaDocs
